tests/testthat/test_GLA_pipeline.R

```diff
@@ -0,0 +1,124 @@
+
+  #Individual AFE3038 - uncertain sex
+  expect_equal(subset(GLA_data, IndvID == "AFE3038")$Sex_calculated, "C") # Should be conflicted
+  expect_equal(subset(GLA_data, IndvID == "AFE3038")$Species, "CYACAE") # Should be conflicted
+  expect_equal(subset(GLA_data, IndvID == "AFE3038")$BroodIDLaid, NA_character_) # Should be NA
+  expect_equal(subset(GLA_data, IndvID == "AFE3038")$BroodIDFledged, NA_character_) # Should be NA
+  expect_equal(subset(GLA_data, IndvID == "AFE3038")$RingSeason, 2019) # Should be 2015 (seen in two years)
+  expect_equal(subset(GLA_data, IndvID == "AFE3038")$RingAge, "adult") # Should be an adult
+
+})
+
+test_that("Brood_data returns an expected outcome...", {
+
+
+
+
+
+
+})
+
+test_that("Capture_data returns an expected outcome...", {
+
+
+  #Take a subset of only GLA data
+  GLA_data <- dplyr::filter(pipeline_output$Individual_data, PopID %in% "CAS", "GAR", "KEL", "SAL", "SCE") %>%
+    arrange(.data$IndvID, .data$BreedingSeason)
+
+  #Test 1: Female caught as adult
+  #Test the female has the correct number of capture records (2) up to and including 2018
+  expect_equal(nrow(subset(GLA_data, IndvID == "S034047"  & BreedingSeason <= 2018)), 2) # Two records
+  #Test the first capture of the female
+  expect_equal(subset(GLA_data, IndvID == "S034047")$Sex_observed[1], "F") # Female
+  expect_equal(subset(GLA_data, IndvID == "S034047")$LocationID[1], "65") # Nest 65
+  expect_equal(min(subset(GLA_data, IndvID == "S034047")$CaptureDate, na.rm = TRUE), as.Date("2017-04-23")) # First caught 2017-04-23
+
+  #Test 2: Male caught as adult
+  #Test the female has the correct number of capture records (2) up to and including 2020
+  expect_equal(nrow(subset(GLA_data, IndvID == "AFE3005")), 2) # Two records
+  #Test the second capture of the male
+  expect_equal(subset(GLA_data, IndvID == "AFE3005")$Sex_observed[2], "M") # Male
+  expect_equal(subset(GLA_data, IndvID == "AFE3005")$LocationID[2], "545") # Nest 65
+  expect_equal(min(subset(GLA_data, IndvID == "AFE3005")$CaptureDate, na.rm = TRUE), as.Date("2020-04-30"))
+
+  #Test 2: Female caught as adult and at three different nests up to and including 2020
+  #Test the female has the correct number of capture records (3) AXB1206
+  expect_equal(nrow(subset(GLA_data, IndvID == "AXB1206")), 3) # Two records
+  #Test the second capture of the male
+  expect_equal(subset(GLA_data, IndvID == "AXB1206")$Sex_observed[1], "F") # Female
+  expect_equal(subset(GLA_data, IndvID == "AXB1206")$LocationID[2], "543") # Nest 543
+  expect_equal(min(subset(GLA_data, IndvID == "AXB1206")$CaptureDate, na.rm = TRUE), as.Date("2018-05-08"))
+
+})
+
+test_that("Location_data returns an expected outcome...", {
+
+  #Take a subset of only GLA data
+  GLA_data <- dplyr::filter(pipeline_output$Individual_data, PopID %in% "CAS", "GAR", "KEL", "SAL", "SCE") %>%
+    arrange(.data$IndvID, .data$BreedingSeason)
```

Comment:
Be careful when copy/pasting code. IndvID and BreedingSeason don't exist in Location_data

```suggestion
  GLA_data <- dplyr::filter(pipeline_output$Location_data, PopID %in% c("CAS", "GAR", "KEL", "SAL", "SCE"))
```